### PR TITLE
Add SVG utility

### PR DESCRIPTION
**Pull request by bennothommo**
This PR adds an SVG utility that can be used to extract and sanitize an SVG from a given path.

This class is a precursor to allowing full support for SVGs in Winter.

### CODE REVIEW

**Comment by LukeTowers** [comment]
@bennothommo would this make more sense under the Winter\Storm\Parse namespace?

**Comment by bennothommo** [comment]
@LukeTowers it's not really a parser though. It's more a utility, like our `Str` or `Arr` utilities.

File: composer.json [hunk 1]
```diff
@@ -44,7 +44,8 @@
         "league/csv": "~9.1",
         "nesbot/carbon": "^2.0",
         "laravel/framework": "~6.0",
-        "laravel/tinker": "~2.0"
+        "laravel/tinker": "~2.0",
+        "enshrined/svg-sanitize": "^0.14.1"
```
Comment (LukeTowers) [inline composer.json]: Why so restrictive on the version? I think they're on 0.15 now

File: src/Support/Svg.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+{
+    /**
+     * Extracts and sanitizes SVG code from a given file.
+     *
+     * @param string $path The path to the SVG file.
+     * @param boolean $minify Whether to minify the extracted SVG code.
+     * @return string
+     * @throws ApplicationException If no file, or a malformed SVG, is found at the given path.
+     */
+    public static function extract($path, $minify = true)
```
Comment (LukeTowers) [inline src/Support/Svg.php:24]: Why Svg::extract() and not Svg::sanitize()?

Comment (bennothommo) [reply src/Support/Svg.php:24]: The intention of the utility is to provide the (sanitized) SVG markup for use, either directly in template code or to be populated into a file, hence why I went with `extract`.

File: composer.json (hunk 1 above)
Comment (bennothommo) [reply composer.json]: I think that was inserted by `composer require`, I'll fix it up.

**Comment by LukeTowers** [comment]
@bennothommo can you add a PR to the docs for this?

**Comment by bennothommo** [comment]
@LukeTowers https://github.com/wintercms/docs/pull/90

**Pull request merged**